opsagent/README.md

```diff
@@ -0,0 +1,40 @@
+# Ops Agent
+
+With [Dataproc 2.2 image version](https://cloud.google.com/dataproc/docs/concepts/versioning/dataproc-release-2.2), we recommend installing [Google Cloud Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent) to obtain system metrics. 
+
+This initialization action will install Ops Agent on a [Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster and provide similar metrics as the [`--metric-sources=monitoring-agent-defaults`](https://cloud.google.com/dataproc/docs/guides/dataproc-metrics#monitoring_agent_metrics) setting which was supported until Dataproc 2.1.
+[This page](https://cloud.google.com/monitoring/api/metrics_agent#oagent-vs-magent) highlights differences in metric collection between Ops Agent and legacy monitoring agent.
+
+We provide two variants of this initialization action:
+- `opsagent.sh` installs Ops Agent. [By default](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#default), it collects syslogs and system (node) metrics.
+- `opsagent_nosyslog.sh` installs Ops Agent and also specifies a user configuration in order to skip syslogs collection from your cluster nodes. If the user configuration is not specified, Ops Agent will collect syslogs besides the system (node) metrics. You can further customize this configuration to collect logs and metrics from other third-party applications.
+
+If you are looking for backward compatibility w.r.t. Dataproc image versions (2.1 and prior) that did not ingest syslogs from Dataproc clusters and want to retain the same behavior, please use `opsagent_nosyslog.sh`.
```

Comment:
I would be careful about making backward compatibility statements — the Ops Agent's host metrics *are* different than those of the standalone ("legacy") agent. I'd also remove the parentheses around the versions… How about something like:
```suggestion
If you are looking to match the behavior of Dataproc image versions up to 2.1 with `--metric-sources=monitoring-agent-defaults`, which did not ingest syslogs from Dataproc nodes, please use `opsagent_nosyslog.sh`.
```
WDYT?

Reply:
Good one, incorporated.